boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -160,14 +160,73 @@ Flashing
 bootloader and another using the SWD debug port (which requires additional
 hardware).
 
+Flashing using the ROM bootloader requires a special activation pattern,
+which can be triggered by using the BOOT0 pin. The ROM bootloader supports
+flashing via USB (DFU), UART, I2C and SPI, but this document describes the
+UART case only. You can read more about how to enable and use the ROM
+bootloader by checking the application note `AN2606`_ .
+
 Using ROM bootloader:
 ---------------------
 
+Hereafter the documents describes basic steps to perform ArgonKey firmware
+flashing on a Linux PC using UART as communication channel.
+
+A. Connect ArgonKey UART to your Linux PC using, for example, a USB-TTL serial
```

Comment:
We use numbered steps rather than a letter, so change this to ``1.`` etc.  or you can use ``#.`` for auto-numbering.

Reply:
ok